Update to 1.21

SuchBlue opened the pull request.

SuchBlue committed:
Updated mappings to 1.20.1

SuchBlue committed:
1.21 Update!

SuchBlue committed:
added priority

SuchBlue committed:
removed deprecated code

SuchBlue committed:
removed buggy cape editor button

eliancode commented:
Nice PR! I have one question though: Is there a specific reason you edited the `gradlew` file? It's most likely just my stupidity, I was just wondering. Also, you could maybe put it in multiple PRs, like for example `chor: add mc version 1.21 support` and `fix: remove buggy cape editor gui button`.

SuchBlue committed:
1.21.3 release

Mod version updated to 2.0.3

SuchBlue committed:
Update PlayerListEntryMixin.java

SuchBlue committed:
1.21.5 release

I had the worst time debugging this and I used ComparableFuture.supplyAsync() to work around a huge lag spike when capeURL.openStream() is called in PlayerHandler which I feel like is a bit janky but it works 😛

SuchBlue committed:
Update gradle, loom

SuchBlue committed:
1.21.9 release

SuchBlue committed:
26.1 release
